index.bs

```diff
@@ -11117,48 +11115,83 @@ is the concatenation of the [=interface=]’s
     making \[[PreventExtensions]] fail.
 </div>
 
+
 <h4 id="es-constants">Constants</h4>
 
-For each [=exposed=] [=constant=] defined on an [=interface=] |A|,
-there must be a corresponding property.
-The property has the following characteristics:
+[=Constants=] are exposed on [=interface objects=],
+[=legacy callback interface objects=],
+[=interface prototype objects=], and
+on the single object that implements the interface,
+when an interface is declared with the [{{Global}}] [=extended attribute=].
```

Comment:
Hmm. In [Chrome](https://cs.chromium.org/chromium/src/third_party/WebKit/Source/modules/filesystem/WindowFileSystem.idl):

```js
> Object.getOwnPropertyDescriptor(Window, 'PERSISTENT')
{value: 1, writable: false, enumerable: true, configurable: false}
> Object.getOwnPropertyDescriptor(Window.prototype, 'PERSISTENT')
{value: 1, writable: false, enumerable: true, configurable: false}
> Object.getOwnPropertyDescriptor(window, 'PERSISTENT')
undefined
```

This API is apparently nonstandard according to MDN so we don't have to worry about it much, but I haven't been able to find any other constants specified on interfaces declared to be [Global].

Reply:
Yeah, this is just rewording what already exists here.

Reply:
Might be worth a follow-up bug to disallow constants on [Global]s, as part of the general deprecation-ish of constants.

Reply:
Issue filed: https://github.com/heycam/webidl/issues/528.